### DISCUSSION TITLE
Please clarify the differences, relationship between PostalServer `Mail servers` and `domains`.?

### DISCUSSION THREAD

**Discussion by rturk** [postalserver/postal#3143] [Help with using Postal] [answered]
Hi All,
I'm strulging to understand diference, nuances, concepts of `mail server` and its relationship with `domains`

- **What are Mail Servers?** They don’t seem like actual servers, but rather like a slot where I can track messages, deliveries, and bounces. Additionally, with unique users, it seems that a mail server within the PostalServer ecosystem functions more as an allocation slot.???

- **What are Domains?** - Well, domains are self-explanatory, but how does a domain relate to a mail server and, overall, to the whole serve?

**Comment by willpower232** [comment, marked as answer]
> a slot where I can track messages, deliveries, and bounces. Additionally, with unique users, it seems that a mail server within the PostalServer ecosystem functions more as an allocation slot

You got it. Mail Servers are a boundary for both users of the web UI and API and SMTP keys. You can also specify retention settings specifically to a Mail Server so some messages can be kept longer than for other Mail Servers if that is what you need.

> What are Domains? 

By knowing which domains you are using Postal can reject messages being sent for domains it doesn't know about, generate a specific DKIM record to sign the messages, and double check the DNS records you have created.

If you need a domain to apply to multiple Mail Servers, you can create it at the Organization level rather than creating it several times within each Mail Server.
